docs/source/tracking.rst

```diff
@@ -722,6 +728,11 @@ then make API requests to your remote tracking server.
         mlflow_set_experiment("/my-experiment")
         mlflow_log_param("a", "1")
 
+Note that to use the artifact storage capabilities of a remote server, the client need a direct access to the artifact store.
+The client is the one pushing directly artifacts to the artifact storage, and not through the tracking server.
+How to setup credentials for the artifact storage depends on the :ref:`Artifact Stores <artifact-stores>` technology setup for the server.
+Same holds for the definition of write options, like giving other users (e.g. the Tracking Server UI) permissions to read the artifacts.  
+Note that some technology choices impose that credential will transit in clear text on the network.
```

Comment:
Could you elaborate this line? (include in what situation will the credentials be sent over the network in clear text)

Reply:
This is true at least for the FTP server, I admit I did not think all options thoroughly, so I thought this would be enough.

About the comment: did you want me to elaborate here or within the PR itself ?

Reply:
Within the PR please. I think the docs readers will have the same question I do. (and if you don't know the answer / can't find it easily, feel free to remove this line)